docs/ng-install.md

```diff
@@ -0,0 +1,201 @@
+
+#### Step 2
+
+Add a `default` export at the bottom with the angular2 metadata that your library provides
+For example:
+[https://github.com/NathanWalker/ng2-cli-test-lib/blob/master/ng2-cli-test-lib.ts](https://github.com/NathanWalker/ng2-cli-test-lib/blob/master/ng2-cli-test-lib.ts)
+
+#### Step 3
+
+At the time of writing, we still need a bundler script to create bundles that will then be included into our app.
+Here is an example of a bundler script which uses `systemjs-builder`, which we tested and works well.
+````js
+var pkg     = require('./package.json');
+var path    = require('path');
+var Builder = require('systemjs-builder');
+var name    = pkg.name;
+
+var builder = new Builder();
+var config = {
+  baseURL: '.',
+  transpiler: 'typescript',
+  typescriptOptions: {
+    module: 'cjs'
+  },
+  map: {
+    typescript: './node_modules/typescript/lib/typescript.js',
+    angular2: path.resolve('node_modules/angular2'),
+    rxjs: path.resolve('node_modules/rxjs')
+  },
+  paths: {
+    '*': '*.js'
+  },
+  meta: {
+    'node_modules/angular2/*': { build: false },
+    'node_modules/rxjs/*': { build: false }
+  },
+};
+
+builder.config(config);
+
+builder
+.bundle(name, path.resolve(__dirname, 'bundles/', name + '.js'))
+.then(function() {
+  console.log('Build complete.');
+})
+.catch(function(err) {
+  console.log('Error', err);
+});
+````
+
+Just include this script in the root of your app named like `bundler-script.js` and run it before publishing your library to `npm`.
+
+Note that when TypeScript version 1.8 is out of `beta`, this script will no longer be needed and this documentation will be updated.
+
+#### Step 4
+
+Bump version, compile and republish.
+
+#### Step 5
+
+There is no step 5. It's that easy. Ok wait a second, what?!
+Let's break down the entry file mentioned above.
+
+```
+// We import these so we can reference them in the 'default' export below
+import {TestDirective} from './src/app/directives/test.directive';
+import {TestService} from './src/app/providers/test.provider';
+import {TestService2} from './src/app/providers/test2.provider';
+import {TestPipe} from './src/app/pipes/test.pipe';
+import {TestStyles} from './src/app/test.styles';
+
+// We export everything we want developers to be able to use manually
+export * from './src/app/directives/test.directive';
+export * from './src/app/providers/test.provider';
+export * from './src/app/providers/test2.provider';
+export * from './src/app/pipes/test.pipe';
+export * from './src/app/test.styles';
+
+// This is the magic.
+// Provides a standard way to export your library to allow angular-cli to help developers setup your library
+// Please note: keys are optional. Your library can provide any metadata is provides.
+export default {
+  directives: [TestDirective],
+  pipes: [TestPipe],
+  providers: [TestService, TestService2],
+  styles: TestStyles.styles(),
+  styleUrls: ['src/app/css/test.css']
```

Comment:
@jkuri curious your thoughts here?  We could leave `styleUrls` out of the documentation if there's a problem. My hope was that it could find any/all the `.css` files listed in `styleUrls` and properly place them according to the way the user had chosen to inject the library during auto-inject.